stylesheets/documentation.css

```diff
@@ -400,6 +400,36 @@ pre {
     background: var(--RATINGS-SECTION)
 }
 
+/* Snackbars */
+.simple-snackbars {
+    background: var(--SIMPLE-SNACKBARS-SECTION);
+    padding: 1rem;
+    border-radius: 1rem;
+}
+
+.snackbars-simple {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0px;
```

Comment:
I think the gap is by default set zero, you don't need to mention it.

Reply:
Yes, it is by default 0. However, the component showcase section of my documentation, in general, has a gap that is being overridden here.